packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -397,6 +397,68 @@ describe('<Tooltip />', () => {
     });
   });
 
+  describe('prop: PopperProps', () => {
+    it('should pass PopperProps to Popper Component', () => {
+      const wrapper = mount(<Tooltip {...defaultProps} PopperProps={{ item: 'value' }} />);
```

Comment:
What do you think of using the `render` API instead (createClientRender)?

Reply:
I spent over 45 minutes attempting to use createClientRender, sadly I don't have anymore time to work on this. I propose that we go with the unit tests that I've provided, and feel free to update them yourself if you'd rather use that approach.

Reply:
We are progressively moving away from enzyme, I think that any new tests using it should be a a no go.

Reply:
We might look into it.